DQM/L1TMonitor/python/L1TCaloLayer1Summary_cff.py

```diff
@@ -1,22 +1,16 @@
 import FWCore.ParameterSet.Config as cms
 
 from L1Trigger.Configuration.CaloTriggerPrimitives_cff import *
-simEcalTriggerPrimitiveDigis.Label = 'ecalDigis'
-simHcalTriggerPrimitiveDigis.inputLabel = cms.VInputTag(
-    cms.InputTag('hcalDigis'),
-    cms.InputTag('hcalDigis')
-)
-simHcalTriggerPrimitiveDigis.inputUpgradeLabel = cms.VInputTag(
-            cms.InputTag('hcalDigis'),
-            cms.InputTag('hcalDigis')
-)
-
 from L1Trigger.L1TCaloLayer1.simCaloStage2Layer1Summary_cfi import *
 from L1Trigger.L1TCaloLayer1.simCaloStage2Layer1Digis_cfi import *
 from DQM.L1TMonitor.L1TCaloLayer1Summary_cfi import *
 
-simCaloStage2Layer1Summary.caloLayer1Regions = cms.InputTag("caloLayer1Digis", "")
-simCaloStage2Layer1Digis.ecalToken = cms.InputTag("ecalDigis", "EcalTriggerPrimitives")
-simCaloStage2Layer1Digis.hcalToken = cms.InputTag("hcalDigis", "")
+dqmSimCaloStage2Layer1Summary = simCaloStage2Layer1Summary.clone(
+    caloLayer1Regions = cms.InputTag("caloLayer1Digis", ""),
+)
+dqmSimCaloStage2Layer1Digis = simCaloStage2Layer1Digis.clone(
+    ecalToken = cms.InputTag("ecalDigis", "EcalTriggerPrimitives"),
+    hcalToken = cms.InputTag("hcalDigis", "")
+)
```

Comment:
They are not, because I am making every mistake I can. Thanks for the catch.

Reply:
Hi Andrew, do you need to define a new instance for the `simCaloStage2Layer1Digis` for this? Can you make use of `valCaloStage2Layer1Digis` from https://github.com/cms-sw/cmssw/blob/master/DQM/L1TMonitor/python/L1TStage2Emulator_cff.py#L10 ?

Reply:
@eyigitba I think in this case it isn't possible, since my student put all of these plots (even those that touch on the emulator) into a data directory. I think there exist DQM configurations where the wider emulation that you've linked are not guaranteed to be run.

In the long run, these plots should be disentangled, and no emulation should be used in this configuration, but for now, guaranteeing the relatively low overhead of this emulation this way is preferable.